.travis.yml

```diff
@@ -1,3 +1,10 @@
+os:
+  - linux
+  - osx
+sudo: false
+dist: trusty
```

Comment:
I know `trusty` is LTS until 2020 but might as well use the latest. So please change this to `bionic`.